common/types_spec.go

```diff
@@ -330,6 +409,27 @@ func BuildHeaderSubmissionV3(payload *VersionedSubmitBlockRequest) (*optimisticv
 
 	versionedPayload := &builderApi.VersionedExecutionPayload{Version: payload.Version}
 	switch payload.Version {
+	case spec.DataVersionFulu:
+		versionedPayload.Fulu = payload.Fulu.ExecutionPayload
+		header, err := utils.PayloadToPayloadHeader(versionedPayload)
+		if err != nil {
+
+			return nil, err
+		}
+		return &optimisticv3.HeaderSubmissionV3{
+			URL: []byte{},
+			Submission: &optimisticv3.VersionedSignedHeaderSubmission{
+				Version: spec.DataVersionFulu,
+				Electra: &optimisticv3.SignedHeaderSubmissionElectra{
```

Comment:
is it supposed to be Fulu right ? i assume update this after optimisticv3 changes